### ISSUE TITLE
Training the Model With the Movie ID and Label

### ISSUE THREAD

**Issue by sagedemage** [sagedemage/movie_recommendation_system#7]
It is recommended to train a model on the dataset's movie ID and label. This would help improve the accuracy of the program.

For example:
```
outputs = model(movie_id)
loss = loss_fn(outputs, movie_id)
```
to
```
outputs = model(movie_ids)
loss = loss_fn(outputs, labels)
```
## Resource
- https://pytorch.org/tutorials/beginner/introyt/trainingyt.html#the-training-loop

**Comment by sagedemage** [comment]
```
git fetch origin
git checkout 7-training-the-model-with-the-movie-id-and-label
```
